contrib/snyk/search.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+rpm -qa | grep -i "wget\|cpython\|gnutls\|glib"
```

Comment:
The search string could be passed as a delimited string argument (like a , separated).